app.yaml

```diff
@@ -32,6 +32,7 @@ handlers:
 - url: /build
   static_dir: build
   secure: always
+  expiration: 30d
   http_headers:
     Cache-Control: 'public, max-age=2592000'
```

Comment:
I wondered that too. Probably not, but I ultimately decided to leave it in because it doesn't hurt, and also to maintain parity with the other stanzas.